raiden/ui/web/src/app/components/token-network/token-network.component.html

```diff
@@ -14,6 +14,7 @@
             <div fxLayout="row"
                  fxLayoutAlign="end" fxFlex="50">
                 <button mat-icon-button color="accent" (click)="showRegisterDialog()"
+                        [disabled]="main"
```

Comment:
You may proxy the getter it in the component though, to access from the template

Reply:
But, anyway, don't simply disable the button, people will start asking. Hide or remove it from the DOM (with *if)